src/auth/auth.ts

```diff
@@ -121,6 +124,20 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> implements BaseAuthI
       .then((decodedIdToken: DecodedIdToken) => {
         // Whether to check if the token was revoked.
         if (checkRevoked || isEmulator) {
+          // Whether user has been disabled.
+          if (checkRevoked) {
+            const uid = decodedIdToken.uid;
```

Comment:
I think we should handle this in `verifyDecodedJWTNotRevoked()`. Otherwise we will end up calling `getUser()` twice (i.e. 2 RPCs).